frappe/utils/image.py

```diff
@@ -17,3 +22,81 @@ def resize_images(path, maxdim=700):
 					im.save(os.path.join(basepath, fname))
 
 					print("resized {0}".format(os.path.join(basepath, fname)))
+
+def process_thumbnail(path, commands):
+
+	if ('.' not in path):
+		return False
+
+	extn = path.rsplit('.', 1)[-1]
```

Comment:
This is only for readability (and maybe some additional reliability), but you could use `os` for this:

```suggestion
	filename, extn = os.path.splitext(path)
	extn = extn.replace(".", "")
```

Reply:
Copied this from frappe codebase, but alas doesn't os.path.splitext split on every dot? would have to add extra logic to capture the very last extension

Reply:
It would actually only pick up the last one:

```python
>>> os.path.splitext("/path/to/file.tar.gz")
('/path/to/file.tar', '.gz')
```

Reply:
Sweet, updating it